Irubido/bips integration v4.5

irubido opened the pull request.

irubido committed:
integration tests flask_10.23.0

irubido committed:
Merge branch 'main' of github.com:MetaMask/test-snaps into irubido/bips-integration_v4.5

irubido committed:
dappetter v4.1.1, cache metaam

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ChainSafe/test-snaps?pullRequest=4) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ChainSafe/test-snaps?pullRequest=4) before we can accept your contribution.<br/>**4** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: irubido<br/>:white_check_mark: Mrtenz<br/>:white_check_mark: BeroBurny<br/>:white_check_mark: FrederikBolding<br/>:x: hmalik88<br/>:x: bowensanders<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ChainSafe/test-snaps?pullRequest=4) it.</sub>

irubido committed:
lint fix

irubido committed:
dappeteer 5

irubido committed:
pull changes, revert

irubido committed:
bip32 dappeteer5 error

hmalik88 committed:
Update snaps packages (#141)

* update packages

* update yarn.lock

* update manifests

bowensanders committed:
Remove all usage of `snap_confirm` (#122)

* changes to the test snaps

* removed confirm totally

* switched dialog port to 8001

* linting fix

* yarn.lock changed

* remove cronjob test

* update packages and key derivation snaps

* update manifests

* remove confirm

* update yarn.lock

* Revert "remove cronjob test"

This reverts commit cc0f8870356d4beedd9dcb8aecc11b31515cec36.

* add cronjob import

* update type

* Update packages/site/src/pages/index.tsx

---------

Co-authored-by: hmalik88 <hassan.malik@consensys.net>
Co-authored-by: Frederik Bolding <frederik.bolding@gmail.com>

github-actions[bot] committed:
5.0.0 (#143)

* 4.6.5

* update changelogs

* update manifests

* fix site log

* update log entries

* update version bump to major

* fix logs

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: hmalik88 <hassan.malik@consensys.net>

hmalik88 committed:
Bump versions used in the update test (#142)

* bump update snap versions

* correct update versions

* fix snap versions

* change to patch release

github-actions[bot] committed:
5.0.1 (#144)

* 5.0.1

* updated logs, versions, and manifests

* changed grouping title

* fix notification-snap package.json version

* fix rpc log

* update notification manifest

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: hmalik88 <hassan.malik@consensys.net>

FrederikBolding committed:
Stop transpiling BIP-44 snap dependencies (#145)

* Stop transpiling BIP-44 snap dependencies

* Fix lint

github-actions[bot] committed:
5.0.2 (#146)

* 5.0.2

* Update changelogs, manifests and package.jsons

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: Frederik Bolding <frederik.bolding@gmail.com>

Mrtenz committed:
Bump peaceiris/actions-gh-pages to more recent version (#147)

FrederikBolding committed:
Fix dialog type for cronjob snap (#150)

github-actions[bot] committed:
5.0.3 (#151)

* 5.0.3

* Update changelogs, manifests and package.json

* Update shasums

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: Frederik Bolding <frederik.bolding@gmail.com>

FrederikBolding committed:
Stop using `wallet_requestSnaps` inside snaps (#152)

* Stop using wallet_requestSnaps inside snaps

* Fix lint

github-actions[bot] committed:
5.1.0 (#153)

* 5.1.0

* Update changelogs, package.jsons and manifests

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: Frederik Bolding <frederik.bolding@gmail.com>

FrederikBolding committed:
Patch luxon to fix LavaMoat issue (#154)

* Patch luxon to fix LavaMoat issue

* Update manifests

github-actions[bot] committed:
5.1.1 (#155)

* 5.1.1

* Update changelogs

* Bump version number in snap manifests to 5.1.1

All the snapps have been updated to version 5.1.1. There are no other changes or updates to this commit.

* Bump version to v5.1.1

This commit updates the 'Unreleased' link with the new tag, v5.1.1, for each package's CHANGELOG.md file. This is
a version increment without any code changes.

* Update shasum values in snap.manifest.json files

All snap.manifest.json file's source shasum value has been updated according to
the changes made inside the directories.

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: Maarten Zuidhoorn <maarten@zuidhoorn.com>

Mrtenz committed:
Change version of update snap from 5.0.0->5.0.1 to 5.1.1->5.1.2 (#156)

The version was outdated, so it is necessary to update it to the latest version of the component.

github-actions[bot] committed:
5.1.2 (#157)

* 5.1.2

* Update changelogs, manifests

* Update CHANGELOG version numbers

Update the unreleased version number to 5.1.2 in all the CHANGELOG files to update the version number used in the changelogs prior to the next release.

* Change "Fixed" to "Changed" and specify the update snap version

The version change was not fixing anything, it was just updating the version of an update snap from '5.0.0' to '5.0.1'. The modification was actually changing it to be from '5.1.1' to '5.1.2'.

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: Maarten Zuidhoorn <maarten@zuidhoorn.com>

FrederikBolding committed:
Add WASM test-snap (#158)

* Add WASM test-snap

* Fix lint and manifests

* Fix changelog

* add newline

* Address PR comments

* Update IDs

github-actions[bot] committed:
5.2.0 (#159)

* 5.2.0

* Added wasm test snap

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: bowensanders <bowensanders@gmail.com>

FrederikBolding committed:
Update RPC snap manifest (#163)

* Fix RPC snap manifest

* Add newline

* Update shasum

bowensanders committed:
Create getEntropy test snap (#162)

* created getEntropy snap

github-actions[bot] committed:
5.3.0 (#164)

* 5.3.0

* fixed changelogs et al

* Update packages/rpc/CHANGELOG.md

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: bowensanders <bowensanders@gmail.com>
Co-authored-by: Frederik Bolding <frederik.bolding@gmail.com>

bowensanders committed:
Create test snap for ethereum-provider endowment (#169)

bowensanders committed:
Create a static test-data.json file for network access test (#171)

github-actions[bot] committed:
5.4.0 (#173)

* 5.4.0

* Added ethereum_provider snap

bowensanders committed:
Create networkAccess snap (#168)

github-actions[bot] committed:
5.5.0 (#174)

* 5.5.0

* fixed changelogs

---------

Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: bowensanders <bowensanders@gmail.com>

BeroBurny committed:
Merge remote-tracking branch 'base/main' into irubido/bips-integration_v4.5

# Conflicts:
#	package.json
#	packages/bip32/snap.manifest.json
#	packages/bip44/snap.manifest.json
#	packages/confirm/src/index.ts
#	packages/cronjob/snap.manifest.json
#	packages/dialog/snap.manifest.json
#	packages/getEntropy/jest.config.js
#	packages/getEntropy/test/integration/index.spec.ts
#	packages/insights/snap.manifest.json
#	yarn.lock

BeroBurny committed:
update dappeteer to `5.2.0`

BeroBurny committed:
fix error snap

BeroBurny committed:
update tests

BeroBurny committed:
workflow improvments

BeroBurny committed:
fix lint and comment

BeroBurny committed:
fix `shasum` for `error` snap

BeroBurny committed:
ci fix `prepare-tests` :man_shrugging:

BeroBurny committed:
`@metamask/test-snap-getentropy` missing `--passWithNoTests`
